torchsummary/torchsummary.py

```diff
@@ -6,6 +6,31 @@
 import numpy as np
 
 
+def parse_tuple_list(output, batch_size):
```

Comment:
It would be better if you could provide a docstring here

Reply:
Thanks for your advice. I have fixed it.